src/styles/index.scss

```diff
@@ -1,3 +1,102 @@
 body {
   margin: 0;
+  font-family: Roboto, Arial, sans-serif;
+}
+
+a,
+p {
+  margin: 0;
+  padding: 0;
+}
+
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 36px;
+    padding-left: 3px;
+    padding-right: 3px;
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), avoid using padding for centering the image. It's better to use flexbox or grid for centering.